themes/default/content/blog/introducing-dev-releases/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Introducing Dev Releases"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date and time this file was generated. Dates are used for display and
+# ordering purposes only; they have no effect on whether or when a post is
+# published. To influence the ordering of posts published on the same date, use
+# the time portion of the date value; posts are sorted in descending order by
+# date/time.
+date: 2024-03-27T17:20:44+01:00
+
+# The draft setting determines whether a post is published. Set it to true if
+# you want to be able to merge the post without publishing it.
+draft: false
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or
+# social-media previews. This field is required or the build will fail the
+# linter test. Max length is 160 characters.
+meta_desc: Introducing Dev Releases for the Pulumi CLI and SDKs.  Discover what they are and how they are useful.
```

Comment:
```suggestion
meta_desc: Introducing Dev Releases for the Pulumi CLI and SDKs: early releases of the Pulumi CLI and SDKs to enable customers to access every change as soon as it has been merged, instead of having to wait for the next release.
```

Reply:
Thanks for all the corrections!  Unfortunately this line is too long (we limit it to 160 characters).  What do you think about "Introducing Dev Releases for the Pulumi CLI and SDKs: early releases to enable customers to access changes as soon as they have been merged.", which is 140 characters?

Reply:
Sounds good!